control-plane/pkg/reconciler/trigger/trigger.go

```diff
@@ -61,10 +63,11 @@ type Reconciler struct {
 	*base.Reconciler
 	*FlagsHolder
 
-	BrokerLister    eventinglisters.BrokerLister
-	ConfigMapLister corelisters.ConfigMapLister
-	EventingClient  eventingclientset.Interface
-	Resolver        *resolver.URIResolver
+	BrokerLister         eventinglisters.BrokerLister
+	ConfigMapLister      corelisters.ConfigMapLister
+	serviceAccountLister corelisters.ServiceAccountLister
```

Comment:
Do we want to make this become public by renaming it to `ServiceAccountLister`?

Reply:
As I see for [namespaced broker,](https://github.com/knative-extensions/eventing-kafka-broker/blob/9541bb6f66b643c470291ca2753fee7c1fb6ad5e/control-plane/pkg/reconciler/broker/namespaced_broker.go#L71) it is public.